index.bs

```diff
@@ -2458,12 +2479,12 @@ The [=remote end steps=] with <var ignore>session</var> and |command parameters|
           and height |clip|["<code>height</code>"].
```

Comment:
Should the coordinates of the clip rect not also be converted based on the specified origin?

Reply:
We do this in the next step when we calculate the intersection.